Magento/ruleset.xml

```diff
@@ -5,202 +5,438 @@
-        <severity>6</severity>
+    <rule ref="Generic.PHP.DeprecatedFunctions">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.CodeAnalysis.ForLoopWithTestFunctionCall">
-        <severity>6</severity>
+    <rule ref="Generic.PHP.NoSilencedErrors">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.CodeAnalysis.JumbledIncrementer">
-        <severity>8</severity>
+    <rule ref="Generic.PHP.Syntax">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.CodeAnalysis.UnconditionalIfStatement">
-        <severity>6</severity>
+    <rule ref="Magento.Legacy.MageEntity">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.CodeAnalysis.UnusedFunctionParameter">
-        <severity>6</severity>
+    <rule ref="Magento.NamingConvention.ReservedWords">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.CodeAnalysis.UselessOverridingMethod">
-        <severity>6</severity>
+    <rule ref="Magento.PHP.FinalImplementation">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.Functions.CallTimePassByReference">
+    <rule ref="Magento.PHP.Goto">
         <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.Metrics.CyclomaticComplexity">
-        <severity>8</severity>
-        <type>warning</type>
+    <rule ref="Magento.PHP.ReturnValueCheck">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.Metrics.NestingLevel">
-        <severity>8</severity>
+    <rule ref="Magento.Security.LanguageConstruct">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.PHP.CharacterBeforePHPOpeningTag">
+    <rule ref="Magento.Security.Superglobal.SuperglobalUsageError">
         <severity>10</severity>
-        <exclude-pattern>*.phtml</exclude-pattern>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.PHP.DeprecatedFunctions">
+    <rule ref="Magento.Strings.ExecutableRegEx">
         <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.PHP.NoSilencedErrors.Discouraged">
-        <severity>8</severity>
+    <rule ref="PSR1.Classes.ClassDeclaration">
+        <severity>10</severity>
+        <type>error</type>
     </rule>
-    <rule ref="Generic.PHP.Syntax">
+    <rule ref="PSR2.Files.ClosingTag">
         <severity>10</severity>
+        <type>error</type>
+        <exclude-pattern>*.phtml</exclude-pattern>
     </rule>
-    <rule ref="Magento.Annotation">
-        <severity>6</severity>
+    <rule ref="Squiz.PHP.Eval">
+        <severity>10</severity>
+        <type>error</type>
+    </rule>
+
+    <!-- Severity 9 warnings: Possible security issues. -->
+    <rule ref="Magento.Security.IncludeFile">
```

Comment:
Propose increase severity to 9 as it may cause security issues.

Reply:
I assume you mean "increase to 10". Done.